### PR TITLE
Enhance issues components and tabs, hide Crypto and DS link and task …

### LINKED ISSUE #559: Enhance issues components and tabs, hide Crypto and DS link and tasks
## Changes

* Change the design of the issue input box and button
* Wrap issue input box and button in a form with `onSubmit` handler
* Hide the Crypto and DS link and tasks because they are not working
* Change the design of the tasks tab
* Set fixed size for the task cards
* Replace hard coded values with Constants

### PR DESCRIPTION
**Pull request by sahsisunny**
### Closes #559 
# Enhance issues components and tabs, hide Crypto and DS link and tasks

## What does this PR do?

1. Change the design of the issue input box and button
2. Wrap issue input box and button in a form with `onSubmit` handler
3. Hide the Crypto and DS link and tasks because they are not working
4. Change the design of the tasks tab
5. Set fixed size for the task cards
6. Replace hard coded values with Constants

## What are changes in this PR?

1. Change the design of the issue input box and button
   - [src/styles/issues.module.scss](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-5103cc3588e4b0d4ec08b79cd42a60e3d53229934adf7dd05e165510a926eaad)
2. Wrap issue input box and button in a form with `onSubmit` handler
   - [src/pages/issues/index.tsx](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-305f1fab4c498e4ed3b197310cd59090b4de0125a794ca05c0df441e2be51c7e)
3. Hide the Crypto and DS link and tasks because they are not working
   - [src/components/navBar/index.tsx](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-21d15ad9c85c261b6c0e3ee41acf62d804b0732d2a4de18b5c6844f7b6325395)
   - [__tests__/Unit/Components/Navbar/Navbar.test.tsx](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-107933ecca70c1438d8393f783fd37b812721c435397f4b51547091efda20a2e)
   - [src/components/Layout/index.tsx](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-1f4e0e61b975c394be4798315c8b4ff7ec43ab241d2f0bb2216fbd36f0260f19)
4. Change the design of the tasks tab
   - [src/components/Tabs/Tabs.module.scss](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-fb962c8217aeab11e6d8418df9041821a9da5762741c57704fc3f8cb7c0af903)
5. Set fixed size for the task cards
   - [src/styles/tasks.module.scss](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-fbdb9687baf28b43f61d927d77ec7742fd31bf8dd644ae1812a86293f8ddcb91)
   - [src/pages/mine/index.tsx](https://github.com/Real-Dev-Squad/website-status/pull/557/files#diff-7caa43c4e08bb118aa8ce473513759522a02963dd73f2a6732827468dd225e1c)

### Before - Tabs
image

### After - Tabs
image

### Before  - Issue page
image

### After - Issue page
image
image

### CODE REVIEW

File: src/pages/issues/index.tsx [hunk 1]
```diff
@@ -25,7 +25,13 @@ const SearchField = ({
     loading,
 }: SearchFieldProps) => {
     return (
-        <div className={classNames.searchFieldContainer}>
+        <form
+            className={classNames.searchFieldContainer}
+            onSubmit={(e) => {
+                e.preventDefault();
+                onSearchTextSubmitted();
+            }}
```
Comment (TanishqSingla) [inline src/pages/issues/index.tsx:33]: pass `onSearchTextSubmitted` as callback and prevent default there.

File: src/styles/issues.module.scss [hunk 2]
```diff
@@ -1,6 +1,9 @@
 $link-text: #2563eb;
 $link-bg: #dbeafe;
 $border-color: #d9d9d9;
+$color-blue: #041187;
```
Comment (TanishqSingla) [inline src/styles/issues.module.scss:4]: can we have better name for this variable, so we know what it is meant for

File: src/styles/tasks.module.scss [hunk 3]
```diff
@@ -1,8 +1,13 @@
 .container {
     display: flex;
     flex-direction: column;
-    align-items: center;
+    width: 60%;
```
Comment (TanishqSingla) [inline src/styles/tasks.module.scss:4]: nit: if possible use max width here

File: src/styles/issues.module.scss [hunk 4]
```diff
@@ -43,10 +46,22 @@ $border-color: #d9d9d9;
     padding: 0.25rem 0.75rem;
     font-size: 0.875rem;
     border-radius: 0.25rem;
-    border: 1px solid $border-color;
+    border: none;
+    background-color: $color-disabled;
+
+    &:focus {
+        outline: none;
+    }
```
Comment (TanishqSingla) [inline src/styles/issues.module.scss]: setting outline none to `none`  may not give feedback for users on using keyboard to focus different button.

**Review by TanishqSingla — CHANGES_REQUESTED** [review]

File: src/styles/issues.module.scss (hunk 4 above)
Comment (sahsisunny) [reply src/styles/issues.module.scss]: removed

File: src/styles/issues.module.scss (hunk 2 above)
Comment (sahsisunny) [reply src/styles/issues.module.scss:4]: This is a generic color type and does not contain any context.

File: src/pages/issues/index.tsx (hunk 1 above)
Comment (sahsisunny) [reply src/pages/issues/index.tsx:33]: This function is pass from props the changing the function will change the behaviour at multiple location.

File: src/styles/tasks.module.scss (hunk 3 above)
Comment (sahsisunny) [reply src/styles/tasks.module.scss]: done

**Review by TanishqSingla — APPROVED** [review]

**Review by bhtibrewal — APPROVED** [review]
looks okay to me

**Review by iamYashSinha — APPROVED** [review]
LGTM

**Pull request merged**